tools/build-release-branch.sh

```diff
@@ -107,6 +133,22 @@ function create_new_release_branches {
 		# Create new branch, push to repo
 		git checkout -b $NEW_UNBUILT_BRANCH
 
+		# Updates file target version.
+		read -n1 -p "Would you like to update the version number in files to $TARGET_VERSION? [y/N]" reply
+		if [[ 'y' == $reply || 'Y' == $reply ]]; then
+			# Replace all file contents.
+			sed -r -i "s/Version: .+/Version: ${TARGET_VERSION}/" jetpack.php
+			sed -r -i "s/'JETPACK__VERSION',(\s+)'(.+)'/'JETPACK__VERSION',\1'${TARGET_VERSION}'/" jetpack.php
+			sed -r -i "s/\"version\": \".+\"/\"version\": \"${NPM_TARGET_VERSION}\"/" package.json
+
+			# Commit changed files.
+			git commit -m "update version" jetpack.php package.json
```

Comment:
Do we need any verification (e.g. `git diff` -- "Does this look good? Y to commit".

Reply:
We can totally do that, good idea!